Gemfile

```diff
@@ -2,31 +2,33 @@
 
 source "https://rubygems.org"
 
-ruby "3.2.2"
+source 'https://rubygems.org'
+
+ruby '3.2.2'
```

Comment:
do you know how these were changed? it looks like the " was switched to ' but we don't want that. also the `source` line is duplicated. there is a merge conflict on this file that's related. I'll resolve it but just wanted to call out these quotes shouldn't be converted like this.

Reply:
I ran rubocop -A to try to fix the lint issues but that affected way more files than i had thought. I can revert those changes

Reply:
ok yeah I think we should revert the ones and only update if it's failing on the code climate build, otherwise we'll end up switching code styles back and forth constantly

Reply:
make sure you pull first, I've already update a few things on the Gemfile